runners/direct-java/build.gradle

```diff
@@ -58,12 +58,14 @@ dependencies {
   compile project(path: ":beam-runners-local-java-core", configuration: "shadow")
   compile project(path: ":beam-runners-java-fn-execution", configuration: "shadow")
   compile project(path: ":beam-sdks-java-fn-execution", configuration: "shadow")
+  compileOnly project(path: ":beam-sdks-java-harness")
```

Comment:
use the `provided` scope.

Reply:
That's what this is, as far as I can tell - https://blog.gradle.org/introducing-compile-only-dependencies

Reply:
We have been using `provided` scope in other places and this gets correctly translated when producing the maven jar.

Your correct in that we should migrate to `compileOnly` as the provided scope but until that happens please stick with `provided` for now. I filed https://issues.apache.org/jira/browse/BEAM-4395 for the migration to `compileOnly`